docs: RFQ UX changes (#212)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/buying/request-for-quotation.md

2. Select the Supplier, click on the supplier again. In this page, click on the + next to 'Supplier Quotation'. A new Supplier Quotation page will be opened, user has to enter the quantity, rate and submit it.

    ![Supplier Quotation from Supplier]({{docs_base_url}}/assets/img/buying/supplier-quotation-from-sup.png)

### 4.2 Supplier Quotation from Supplier

1. If a [Contact](/docs/user/manual/en/CRM/contact) is created for the Supplier and an email address is associated with the Contact, the Contact details and the email address will be fetched on selecting the Supplier. Create a Contact and email address if not present already.

2. Click on **Tools > Send Emails to Suppliers**.

>     **If the Supplier's account is not present**: The system will create the Supplier's account and send details to the Supplier. The Supplier will need to click on the link(Password Update) present in the email. After the password update, the Supplier can access their portal with the 'Request for Quotation' form. The Supplier will be created as a Website User.

Reviewer: ```suggestion
    **If the Supplier's account is not present**: The system will create the Supplier's account and send details to the Supplier. The Supplier will need to click on the link (Password Update) present in the email. After the password update, the Supplier can access their portal with the 'Request for Quotation' form. The Supplier will be created as a Website User.
```

    ![Supplier email if account not present]({{docs_base_url}}/assets/img/buying/supplier-email-with-update-password.png)

>     **If Supplier's account is present**: System will send Request for Quotation link to Supplier, Supplier has to login using his credentials to view the Request for Quotation form on portal.

Reviewer: ```suggestion
    **If Supplier's account is present**: System will send Request for Quotation link to the Supplier. The Supplier has to log in using his credentials to view the Request for Quotation form on the portal.
```

    ![Supplier email if account present]({{docs_base_url}}/assets/img/buying/supplier-email-normal.png)

3. Either way, when the Supplier logs in, the following screen will be shown to them. From here they can send you a quotation:

    ![Supplier Quotation Screen]({{docs_base_url}}/assets/img/buying/rfq-supplier-quotation.png)

>     The supplier has to enter the amount and notes (payment terms) on the form and click on Submit. In the Quotations section, previous quotations will be visible.

Reviewer: ```suggestion
    The Supplier has to enter the amount and notes (payment terms) on the form and click on Submit. In the Quotations section, previous quotations will be visible.
```

> 4. On submission, ERPNext will create a Supplier Quotation (draft mode) against the Supplier. The user has to review the Supplier Quotation and submit it. When all items from the Request for Quotation have been quoted by a Supplier, the quote status is updated to 'Received' in the 'Suppliers' table of Request for Quotation.

Reviewer: ```suggestion
4. On submission, ERPNext will create a Supplier Quotation (draft mode) against the Supplier. The user has to review the Supplier Quotation and submit it. When all the items from the Request for Quotation have been quoted by a Supplier, the quote status is updated to 'Received' in the 'Suppliers' table of the Request for Quotation.
```
